doc/reference/reference_lua/box_events/watch.rst

```diff
@@ -0,0 +1,37 @@
+.. _box-watch:
+
+================================================================================
+box.watch()
+================================================================================
+
+..  function:: box.watch(key, func)
+
+    Subscribe to events broadcast by a local host.
+
+    :param string key: a key name of an event to subscribe to
+    :param function func: a callback to invoke when the key value is updated
+
+    :return: a watcher handle. The handle consists of one method -- ``unregister()``, which unregisters the watcher.
+
+    To read more about watchers, see the `Functions for watchers <box-watchers>` section.
+
+    ..  note::
+
+        Keep in mind that garbage collection of a watcher handle doesn't lead to the watcher's destruction.
+        In this case, the watcher remains registered.
+        It is okay to discard the result of ``watch`` function if the watcher will never be unregistered.
+
+    **Example:**
+
+    ..  code-block:: lua
+
```

Comment:
I think that the example should be with netbox:

1. a client does netbox.watch
2. a servers does box.broadcast

Reply:
And notice that server and client can be placed to one host.